### DISCUSSION TITLE
Screenshot valuable drops to screenshot pickpocket loot

### DISCUSSION THREAD

**Discussion by absywabsy** [runelite/runelite#16217] [Ideas]
Pickpocket loot above the threshold (e.g. blood shards) are not screenshotted when pickpocketed. Is it possible to include the screenshotting of this when the pickpocket returns with loot higher than the threshold entered under Screenshot > Screenshot Valuable Drops > Valuable Threshold?

https://i.imgur.com/wA8MQ8G.png

**Comment by Enriath** [comment]
That feature works off of the vanilla `Valuable Drops` system (specifically, the chat message it makes). If pickpocketing doesn't create those messages, then there's not much to do about it.

**Reply by absywabsy** [reply]
I've set the valuable drops threshold value in game to 1gp and when I pickpocket to receive anything, no chat messages are generated.

That's a real shame but thanks for your response.
